config/routes.rb

```diff
@@ -9,7 +9,12 @@
   post "frontend_register/:usermodel"  => "sessions#register", as: :frontend_register
   post "manage/article_visibility/:id" => "manage#article_visibility"
 
-  mount Sidekiq::Web => "/admin/background" if RUBY_VERSION.to_f >= 1.9
+  if RUBY_VERSION.to_f >= 1.9
+    # Make Sinatra and Rails share a session to prevent CSRF errors when using Sidekiq web
+    # interface: https://github.com/mperham/sidekiq/issues/1289#issuecomment-232330804
+    Sidekiq::Web.set(:session_secret, Rails.application.secrets[:secret_key_base])
```

Comment:
Magst du ausprobieren `Sidekiq::Web.set(:session_secret, Rails.application.secrets[:secret_key_base])` nicht auch in einem initializer Platz finden würde. Das sieht hier in den `routes.rb` etwas Fehl am Platz aus